packages/components/src/progress-bar/styles.ts

```diff
@@ -67,7 +76,7 @@ export const Indicator = styled.div< {
 					animationDuration: '1.5s',
 					animationTimingFunction: 'ease-in-out',
 					animationIterationCount: 'infinite',
-					animationName: animateProgressBar,
+					animationName: animateProgressBar( isRTL() ),
```

Comment:
I don't think I've ever thought of using the `isRTL()` check directly in the styled components, I always  used it in the render function and passed it as a prop. Interesting!

Reply:
It felt simpler to me, rather than having to introduce yet another prop. I'm happy to consider changing it if you feel it can be improved.